3737/1/parser.rb

```diff
@@ -0,0 +1,19 @@
+require 'nokogiri'
+require 'open-uri'
+require 'xpath'
+
+# class for getting github page
+class Parser
+  def search_repo(gem_name)
+    url = Nokogiri::HTML(open("http://rubygems.org/gems/#{gem_name}"))
+    check_page('code', url) || check_page('home', url)
+  end
+
+  private
+
+  def check_page(link_id, url)
```

Comment:
method name doesn't reflect what is returned. you are looking for github_url. so it should be reflected in method_name